components/Map/Map.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import L from 'leaflet';
+
+interface MapProps {
+  saveMapState: (arg0: any, arg1: any) => void;
+  width: string;
+  height: string;
```

Comment:
Make optional with defaults set in `defaultProps`

Reply:
Now optional, defaults were already set